src/main/scala/swiss/dasch/api/MonitoringEndpointsHandler.scala

```diff
@@ -24,9 +24,8 @@ case class MonitoringEndpointsHandler(
   val healthEndpoint: ZServerEndpoint[Any, Any] =
     monitoringEndpoints.healthEndpoint
       .zServerLogic(_ =>
-        healthService.check.flatMap {
-          case it if it.isHealthy => ZIO.succeed(it)
-          case _                  => ZIO.fail(ApiProblem.Unhealthy())
+        healthService.check.flatMap { res =>
+          if (res.isHealthy) ZIO.succeed(res) else ZIO.fail(ApiProblem.Unhealthy.from(res))
```

Comment:
FYI, this likely has an ZIO alias, it looks like it might be filterOrElseWith.

Reply:
Thanks, thats it: 7339cd72e80d0789b9a55014febe09ff8bc4c396